tests/unit/Services/ReapplyServiceTest.php

```
@@ -132,6 +201,52 @@ public function testEnsureDirectoryFailureIsReportedAsError(): void
+         * already exists so 'ensureDirectory' is a no-op and the entry is reapplied, proving the loop continues.
+         */
+        $this->seedTracked('deep/config/params.php', "stub\n", "stub\n");
+        $this->seedTracked('top.txt', "stub\n", "stub\n");
+
+        MockerState::addCondition(
+            'yii\\scaffold\\Scaffold',
+            'mkdir',
+            [],
+            false,
+            default: true,
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xepozz\InternalMocker\MockerState' in method 'testEnsureDirectoryFailureSkipContinuesIterationToNextReapplicableEntry'. (undefined)

(StaticAccess)